### DISCUSSION TITLE
Verbiage on closed meetings

### DISCUSSION THREAD

**Discussion by VenturaCntyCentralOffice** [code4recovery/12-step-meeting-list#467] [General]
In a recent update the verbiage on closed meetings was changed. 

> "Closed meetings are for A.A. members only, or for those who have a drinking problem and “have a desire to stop drinking.”

Besides the fact that the statement is semantically incorrect  - you can't use the word ONLY followed by 2 exception. If there are exception then technically it is not ONLY. However, my main issue is with the word AND in this part for those who have a drinking problem **and** “have a desire to stop drinking. While many meetings and websites say AND the word should be OR. You can acknowledge that you have a drinking problem and are not yet at the point you have a desire to stop - you are welcome to come to a closed meeting and check it out. 

The wording, from the 1987 General Service Conference, as it appears on conference approved F-7 is:
  

>  In support of A.A.'s singleness of purpose, attendance at closed meetings is limited to persons who have a desire
>    to stop drinking. If you think you have a problem with alcohol, you are welcome to attend this meeting.

The wording clearly indicates 2 different statement, as indicated by the period.

Sorry to be a little long winded and maybe nitpicky but I would recommend changing the AND to OR. Also, there is nothing about a closed meeting that requires someone to be an A.A member, as long as you agree with one of the statements. 

Kindest Regards,

Matthew C.

**Comment by tech2serve** [comment]
Hi @VenturaCntyCentralOffice!

At the request of other webmasters, we adopted the language used by GSO's Meeting Guide app. Not to say this won't change in the future though.

In the meantime, you can use site customizations (you'll need access to your (child) theme's `functions.php`) to change the language as you wish for your site:

```
	if (function_exists('tsml_custom_types')) {
		tsml_custom_types(array(
			'XYZ' => 'My Custom Type',
		));
	}
```

As long as you use `'C'` in place of `'XYZ'`, the meetings should still correctly display as closed in Meeting Guide.
